libs/perun/components/src/lib/perun-footer/perun-footer.component.html

```diff
@@ -1,22 +1,32 @@
 <footer [ngStyle]="{'background': bgColor}" class="footer-container">
   <div class="footer-columns-container">
-    <div *ngFor="let item of items" class="footer-col mx-auto">
+    <div *ngFor="let item of items" [class.my-auto]="containsLogos" class="footer-col mx-auto">
```

Comment:
Still headers are not at the same position.
![image](https://user-images.githubusercontent.com/70371638/145406867-3e119b3c-ef56-482e-871e-3cfed5c1f615.png)